.github/workflows/test.yaml

```diff
@@ -38,6 +38,17 @@ jobs:
           extraPullNames: holochain-ci
           authToken: ${{ secrets.CACHIX_HOLOCHAIN_WIND_TUNNEL }}
 
+      - uses: actions/cache@v4
+        with:
+          path: |
+            .cargo/bin/
+            .cargo/registry/index/
+            .cargo/registry/cache/
+            .cargo/git/db/
+            target/
+            wasm-target/
+          key: "${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}"
```

Comment:
Two thoughts:
- this could get large, consider [cargo-sweep](https://crates.io/crates/cargo-sweep) `--stamp` before, then `--file` after.
- I've been pretty happy with [Swatinem/rust-cache](https://github.com/marketplace/actions/rust-cache), but maybe that doesn't integrate with nix as well?

Reply:
Ahh yes okay, because Holonix moves the cache into the project I'm not that keen to use the `rust-cache`. If we don't maintain that behavior with the new Holonix then I'd be happy to switch. Taken suggestion 1 for now